acceptance_tests/__init__.py

```diff
@@ -1,7 +1,12 @@
+import logging
 import os
 
 from splinter import Browser
+from structlog import configure
+from structlog.stdlib import LoggerFactory
 
+logging.basicConfig()
+configure(logger_factory=LoggerFactory())
```

Comment:
Why do we need to do this?

Reply:
It's not actually logging anything at the moment. This will fix that

Reply:
Do we need this can't we use one of the --no-capture commands for running the tests? https://pythonhosted.org/behave/behave.html#cmdoption--no-capture

Reply:
Didn't work unless I am missing something?

Reply:
Not sure, I have heard it works